usage/rest-server/src/main/java/org/apache/brooklyn/rest/transform/CatalogTransformer.java

```diff
@@ -91,7 +91,7 @@
         return new CatalogEntitySummary(item.getSymbolicName(), item.getVersion(), item.getDisplayName(),
             item.getJavaType(), item.getPlanYaml(),
             item.getDescription(), tidyIconLink(b, item, item.getIconUrl()),
-            config, sensors, effectors,
+            item.tags().getTags(), config, sensors, effectors,
```

Comment:
The tags are generated at the time the catalog item is created. In the mean time the dependent catalog items may change, leading to changes in the interfaces implemented by this catalog item. Is this acceptable? Could generate them on the fly here.